### PR TITLE
Link to the W3C community page for this group, not just to its join link. Fixes #15.

### LINKED ISSUE #15: More clearly link to the w3c community group page for this CG
Currently the only link to the CG page from this page is the "join" link. I think it would be better to have a clear link so people can go to that pae and find out who's in the group, etc...

### PR DESCRIPTION
**Pull request by hober**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,20 +30,20 @@ <h1>Privacy Community Group</h1>
...
-simply <a href=mailto:erik.anderson@microsoft.com;hober@apple.com;tanvi@mozilla.com>email
-them</a> (if your concern can’t be raised publicly).
+<h2>Participate</h2>
+
+<p>Our <a href=https://www.w3.org/community/privacycg/>W3C page</a>
+contains a list of
+<a href=https://www.w3.org/community/privacycg/participants>current
+participants</a>.
+
+<p>We hope you'll
```
Comment (TanviHacks) [inline index.html]: Switch this with the previous paragraph; first asking folks to participate, then showing the current participants.

Comment (TanviHacks) [reply index.html]: May require a change to the beginning of this sentence as well.

File: index.html [hunk 2]
```diff
@@ -30,20 +30,20 @@ <h1>Privacy Community Group</h1>
 title="Community Group">CG</abbr></a>
 is <a href=charter.html>chartered</a> <q cite=charter.html#mission>to
 incubate privacy-focused web features and APIs to improve user privacy
-on the web through enhanced browser behavior.</q> If that sounds
-interesting to you, please
-<a href=https://www.w3.org/community/privacycg/join>join us</a>!
```
Comment (TanviHacks) [inline index.html]: Keep this text "if that sounds interesting to you, please join us", but link instead to the participants section (if that's possible).

Comment (hober) [reply index.html]: Restored.

File: index.html (hunk 1 above)
Comment (hober) [reply index.html]: Done.

**Comment by hober** [comment]
I've rebased this so it'll cleanly merge. Please review, @TanviHacks & @torgo!

**Pull request merged**